src/core/meshcop/dataset_manager.cpp

```diff
@@ -788,6 +788,20 @@ void PendingDatasetManager::HandleDelayTimer(void)
             ExitNow();
         }
     }
+#if OPENTHREAD_CONFIG_AMEND_PARTIAL_PENDING_OPERATIONAL_DATASET
+    if (dataset.GetTlv(Tlv::kNetworkKey) == nullptr)
+    {
+        Dataset activeDataset;
+        IgnoreError(Get<ActiveDatasetManager>().Read(activeDataset));
```

Comment:
Nit: Empty line after variable declaration ([per style guide](https://github.com/openthread/openthread/blob/f14d7264dae20d99702f0ba128837a573dd3c8fc/STYLE_GUIDE.md#white-space)).
```suggestion
        Dataset activeDataset;
        
        IgnoreError(Get<ActiveDatasetManager>().Read(activeDataset));
```